com.mandelbulber.Mandelbulber.yaml

```diff
@@ -0,0 +1,117 @@
+runtime-version: '5.15'
+command: mandelbulber2
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --filesystem=home
+cleanup:
+  #- /include
+  #- /lib/cmake
+  #- /share/doc
+  #- /share/applications
+  #- *.la
+  #- *.a
+
+modules:
+  #- name: libpng12
+    #rm-configure: true
+    #config-opts:
+      #- --disable-static
+      #- --enable-maintainer-mode
+      #- --with-libpng-compat
+    #sources:
+      #- type: archive
+        #url: https://sourceforge.net/projects/libpng/files/libpng12/1.2.59/libpng-1.2.59.tar.xz
+        #sha256: b4635f15b8adccc8ad0934eea485ef59cc4cae24d0f0300a9a941e51974ffcc7
+      #- type: shell
+        #commands:
+          #- cp -p /usr/share/automake-*/config.{sub,guess} .;
+          
+  - name: gsl
+    config-opts:
+      - --disable-static
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/gsl/gsl-2.6.tar.gz
+        sha256: b782339fc7a38fe17689cb39966c4d821236c28018b6593ddb6fd59ee40786a8
+
+  - name: opencl-headers
+    buildsystem: simple
+    build-commands:
+      - cp -av CL /app/include
+    sources:
+      - type: archive
+        url: https://github.com/KhronosGroup/OpenCL-Headers/archive/refs/tags/v2020.06.16.tar.gz
+        sha256: 2f5a60e5ac4b127650618c58a7e3b35a84dbf23c1a0ac72eb5e7baf221600e06
+  
+  #- name: ocl-icd
+    #sources:
+      #- type: archive
+        #url: https://github.com/OCL-dev/ocl-icd/archive/refs/tags/v2.2.14.tar.gz
+        #sha256: 46df23608605ad548e80b11f4ba0e590cef6397a079d2f19adf707a7c2fbfe1b
+  
+  - name: opencl-cpp-headers
+    buildsystem: cmake
+    config-opts:
+      - -DBUILD_DOCS:BOOL=OFF
+      - -DBUILD_TESTS:BOOL=OFF
+      - -DBUILD_TESTS:BOOL=OFF
+    sources:
+      - type: archive
+        url: https://github.com/KhronosGroup/OpenCL-CLHPP/archive/refs/tags/v2.0.12.tar.gz
+        sha256: 20b28709ce74d3602f1a946d78a2024c1f6b0ef51358b9686612669897a58719
+
+  - name: lzo
+    config-opts:
+      - --enable-shared
+      - --disable-static
+    sources:
+      - type: archive
+        url: http://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz
+        sha256: c0f892943208266f9b6543b3ae308fab6284c5c90e627931446fb49b4221a072
+      
+    # cannot use qmake build system, target make install not found  
+  - name: mandelbulber
+    buildsystem: simple
+    build-commands:
+      - cd mandelbulber2/qmake && qmake mandelbulber-opencl.pro
+      - cd mandelbulber2/qmake && make all
+      - cd mandelbulber2 && install -Dm0755 qmake/mandelbulber2 /app/bin/mandelbulber2
+      - mkdir -p app/share/mandelbulber2/formula/ app/share/mandelbulber2/opencl/
+      #- install -Dm0644 mandelbulber2/deploy/share/mandelbulber2/icons/mandelbulber.png app/share/pixmaps/mandelbulber.png
+      - install -Dm0644 mandelbulber128.png app/share/pixmaps/mandelbulber.png
+      - install -Dm0644 mandelbulber128.png app/share/app-info/icons/flatpak/128x128/com.mandelbulber.Mandelbulber.png
```

Comment:
`install -Dm644 mandelbulber128.png /app/share/icons/hicolor/128x128/apps/mandelbulber.png`

Reply:
Thanks for the help, but it does not seems to work. Any ideas?

Reply:
I've sent a suggestion to https://github.com/kevinsmia1939/flathub/pull/1.
If you want to keep using the simple buildsystem, you need to add `rename-icon: mandelbulber`.
(also: `tags` should be `tag` in git source section)